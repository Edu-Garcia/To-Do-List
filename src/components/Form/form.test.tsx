import { render } from '@testing-library/react'
import { BrowserRouter } from 'react-router-dom';
import { Form } from '.'
import '@testing-library/jest-dom';

describe('Form Test', () => {
  it('snapshot test form component', () => {
    const { container } = render(
      <BrowserRouter>
        <Form addTask={jest.fn()} />
      </BrowserRouter>
    );

    expect(container).toMatchSnapshot();
  });
});