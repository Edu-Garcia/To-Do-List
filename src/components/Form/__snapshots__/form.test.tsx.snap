// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form Test snapshot test form component 1`] = `
<div>
  <form>
    <div>
      <label
        for="title"
      >
        Título
      </label>
      <input
        id="title"
        name="title"
        placeholder="Adicione um título"
        required=""
        type="text"
        value=""
      />
    </div>
    <div>
      <label
        for="description"
      >
        Descrição
      </label>
      <input
        id="description"
        name="description"
        placeholder="Adicione uma descrição"
        type="text"
        value=""
      />
    </div>
    <button
      class="button undefined"
      type="submit"
    >
      Enviar
    </button>
  </form>
</div>
`;
